File: fstp-core-bo/src/main/java/com/purefun/fstp/core/bo/QueryRequestBO.java
package com.purefun.fstp.core.bo;

import java.io.Serializable;

import com.purefun.fstp.core.tool.UUID;
import com.purefun.fstp.core.tool.fstpbo;

@fstpbo(boid = 5L, destination = "fstp.core.manager.queryrequest")
public class QueryRequestBO extends BaseBO  {
	
	public String requestServiceName = "";
	
	public String respondServiceName = "";
	
	public String querytopic = "";
	
	public String tempTopic = "";	
}

File: fstp-core-bo/python/com/purefun/fstp/core/bo/model/QueryRequestBO.py
import uuid

class QueryRequestBO(object):

    def __init__(self):
        self.uuid = str(uuid.uuid1())
        self.boid = 5L
        self.destination = "fstp.core.manager.queryrequest"
        self.requestServiceName = ''
        self.respondServiceName = ''
        self.querytopic = ''
        self.tempTopic = ''
